utils/dev-server.js

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
There's already https://github.com/GoogleChrome/sw-helpers/blob/master/gulp-tasks/serve.js

It would be good to only keep one of them. Is there anything this does that the `gulp serve` task doesn't accomplish? If we keep this, can you modify` gulp serve` so that it starts up this script?